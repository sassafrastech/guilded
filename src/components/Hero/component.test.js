import React from 'react';
import renderer from 'react-test-renderer';

import Component from './component';

it('renders', () => {
  const tree = renderer.create(<Component />);
  expect(tree.toJSON()).toMatchSnapshot();
});
