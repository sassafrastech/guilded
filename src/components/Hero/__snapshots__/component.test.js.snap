// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div
  className="makeStyles-container-1"
>
  <img
    alt="Guilded Logo"
    className="makeStyles-logo-2"
    src="logo.svg"
  />
  <div
    className="makeStyles-textContainer-3"
  >
    <div
      className="makeStyles-dash-4"
    />
    <h3
      className="MuiTypography-root makeStyles-title-5 MuiTypography-h3"
    >
      True empowerment for freelancers
    </h3>
    <h6
      className="MuiTypography-root makeStyles-subtitle-6 MuiTypography-h6"
    >
      Guilded is a freelancer worker cooperative that focuses on the agency, decision-making and economic power of the freelancer. We offer invoicing, tax preparation, health care and advocacy support so that you can focus on doing what you do best. Scroll down to learn more.
    </h6>
    <div
      className="makeStyles-buttonContainer-7"
    >
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-8 MuiButton-containedPrimary MuiButton-containedSizeLarge MuiButton-sizeLarge"
        href="https://guildedcoop.typeform.com/to/vAo4vj"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          Support Guilded
        </span>
      </a>
      <a
        aria-disabled={false}
        className="MuiButtonBase-root MuiButton-root MuiButton-contained makeStyles-button-8 MuiButton-containedSecondary MuiButton-containedSizeLarge MuiButton-sizeLarge"
        href="https://guildedcoop.typeform.com/to/jFmHAFvv"
        onBlur={[Function]}
        onDragLeave={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
      >
        <span
          className="MuiButton-label"
        >
          I am Interested
        </span>
      </a>
    </div>
  </div>
</div>
`;
